lib/simplecov/configuration.rb

```diff
@@ -318,16 +326,21 @@ def maximum_coverage_drop(coverage_drop = nil)
     # Default is 0% (disabled)
     #
     def minimum_coverage_by_file(coverage = nil)
-      minimum_possible_coverage_exceeded("minimum_coverage_by_file") if coverage && coverage > 100
-      @minimum_coverage_by_file ||= (coverage || 0).to_f.round(2)
```

Comment:
mental note that we're dropping the `.to_f.round(2)` which was probably there for a reason but I can't seem to find a reason to keep it right now so maybe we removed some unnecessary complexity :rocket: